hooks/emitter/oracle.go

```diff
@@ -208,14 +220,20 @@ func (h *Hook) handleMsgEditOracleScript(
 
 // handleEventRequestExecute implements emitter handler for EventRequestExecute.
 func (h *Hook) handleEventRequestExecute(ctx sdk.Context, evMap common.EvMap) {
+	var gasUsedI interface{}
+	if gasUsedMap, ok := evMap[types.EventTypeResolve+"."+types.AttributeKeyGasUsed]; ok {
```

Comment:
the name `map` doesn't align with others. 

Should we write helper functions such as getPrepareGasUsed(evMap), getExecuteGasUsed(evMap)? What do you think?

Reply:
I think no need to have helper functions.